test/conf_include_test.c

```diff
@@ -186,7 +186,7 @@ static int test_check_overflow(void)
     char max[(sizeof(long) * 8) / 3 + 3];
     char *p;
 
-    p = max + sprintf(max, "0%ld", LONG_MAX) - 1;
+    p = max + BIO_snprintf(max, sizeof (max), "0%ld", LONG_MAX) - 1;
```

Comment:
nit: should there be a space after `sizeof`?

Reply:
This extra space was a glitch - removed.